lib/zora/score/getRewardScore.tsx

```diff
@@ -1,9 +1,7 @@
-import { MAX_REWARD_SCORE } from '@/lib/consts';
-import { formatEther } from 'viem';
+import { ETH_TO_SPARKS, MAX_REWARD_SCORE } from '@/lib/consts';
 
 export function getRewardScore(totalRewards: number) {
-  const totalRewardsInBigInt = BigInt(totalRewards);
-  const totalRewardsInETH = parseFloat(formatEther(totalRewardsInBigInt));
-  const rewardScore = (totalRewardsInETH / MAX_REWARD_SCORE) * 100;
+  const totalRewardsInETH = totalRewards / ETH_TO_SPARKS;
```

Comment:
Instead of converting the sparks to ETH to handle the conversion (decimal arithmetic)
Would this code be simpler if
MAX_REWARD_SCORE const was denominated in `sparks` instead of `ETH` for direct comparison without constant denomination conversion (whole number arithmetic)